webcomponents-lite-debug.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+(function() {
```

Comment:
I'm afraid of calling this the "debug" version of "lite", since it uses a completely different strategy to load in the scripts. `document.write` will cause all of the loaded scripts to be evaluated in the order that they are written, while the new "lite" strategy causes scripts to be evaluated in the order they load. An error caused by unexpected loading order in "lite" will actually be obscured by this "debug" version.